### PR TITLE
Add corcel extension

### PR DESCRIPTION
**Pull request by AkinAguda**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/corcel/src/actions/add-or-remove-fav-img.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                showToast({ title: "Added to favourites successfully!", style: Toast.Style.Success });
+              })
+              .catch(() => {
+                showToast({ title: "Failed to add image to favourites.", style: Toast.Style.Failure });
+              });
+          }}
+        />
+      ) : (
+        <Action
+          title="Remove from Favourites"
```
Comment (pernielsentikaer) [inline extensions/corcel/src/actions/add-or-remove-fav-img.tsx:36]:
```suggestion
          title="Remove from Favourites"
          style={Action.Style.Destructive}
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Could you remove `metadata/0.png` since it doesn't have any contextual value

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/corcel/src/actions/add-or-remove-fav-img.tsx (hunk 1 above)
Comment (AkinAguda) [reply extensions/corcel/src/actions/add-or-remove-fav-img.tsx:36]: Added suggestion

**Comment by AkinAguda** [comment]
Screenshot 2024-04-25 at 10 10 37

Raycast lint rules flags your change @pernielsentikaer. I assume this a bug with the lint rule

Updated

**Comment by pernielsentikaer** [comment]
It should also work in the linter, let me follow up on that.

You can have 0.png in the README, just create a folder called images and reference it from that folder if you want 🙂

**Comment by AkinAguda** [comment]
> It should also work in the linter, let me follow up on that.
> 
> You can have 0.png in the README, just create a folder called images and reference it from that folder if you want 🙂

I'm not sure what you mean here. I renamed 0.png to 1.png and all the other images accordingly. I want it to be part of the metadata images (the ones shown on the extension's page on raycast)

It's a screenshot showing all the commands available in this extension. Let me know if my latest commit adding a description for the image in the README is fine now.

Thanks @pernielsentikaer

**Comment by AkinAguda** [comment]
Ahh I see what you mean now

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**